collector/mongos/sharding_topology.go

```diff
@@ -0,0 +1,181 @@
+package collector_mongos
+
+import (
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+var (
+	shardingTopoInfoTotalShards = prometheus.NewGauge(prometheus.GaugeOpts{
+			Namespace:	Namespace,
+			Subsystem:	"sharding",
+			Name:		"shards_total",
+			Help:		"Total # of Shards in the Cluster",
+	})
+	shardingTopoInfoDrainingShards = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
No I was talking about  +			Name:		"shards_draining_total", (I had highlighted the  line that was the start of the function, so was not talking about shards_total. Remoing comment we will fix this later as you added the beta tag